docs/getting-started/setup.md

```diff
@@ -8,34 +8,31 @@ menu:
 
 ## Installation
 
-### Via WordPress.org (easy)
-
-You can grab the all-things-included plugin at [WordPress.org](http://wordpress.org/plugins/timber-library/) either through the WordPress site or through the Plugins menu in the backend. Then skip ahead to [using the starter theme](#use-the-starter-theme).
-
-### Via GitHub (for developers)
-
-The GitHub version of Timber requires [Composer](https://getcomposer.org/download/). If you'd prefer one-click installation, you should use the [WordPress.org](https://wordpress.org/plugins/timber-library/) version.
+You can install Timber through [Composer](https://getcomposer.org/download/).
 
 ```shell
 composer require timber/timber
 ```
 
-If your theme is not setup to pull in Composer’s autoload file, you will need to add the following at the top of your `functions.php` file: 
+You can choose yourself where in your project you want to include Timber. Some developers prefer to have Timber installed as a theme dependency, so they would run this command from the theme root. But it’s also possible to use Timber as a WordPress dependency, which means you would run the command above from the WordPress root.
```

Comment:
I think this should be `**Most** developers prefer to have Timber installed as a theme dependency.` It's pretty clear that the other suggestion is more alternative `it's also possible...` I have to confess that I've only ever tested/used it as a theme or plugin dependency — never for the entire WP install

Reply:
Yeah, that’s probably better. I replaced "Some" with "Most".